spec/discovery.md

```diff
@@ -59,6 +59,18 @@ curl $(echo "$urltmpl" | sed -e "s/{name}/$name/" -e "s/{version}/$version/" -e
 
 where _name_, _version_, _os_, and _arch_ are set to their respective values for the image, and _ext_ is either `aci` or `aci.asc` for retrieving an App Container Image or signature respectively.
 
+The client MUST accept only best matching templates. Best matching templates are the one where all the templates labels can be substituted and with the highest number of substituted labels.
```

Comment:
> but in the text you have
> 
> all the templates labels can be substituted and with the highest number of substituted labels

Sorry, I'll trying to reword it:

`
all the templates expressions can be substituted by the client supplied labels and have the highest number of substituted client supplied labels.
`